examples/siro_sandbox/app_states/app_state_fetch.py

```diff
@@ -250,11 +266,15 @@ def _update_help_text(self):
                 text_delta_y=-50,
             )
 
-    def _get_camera_lookat_pos(self):
+    def _get_agent_pose(self):
         agent_root = get_agent_art_obj_transform(
             self.get_sim(), self.get_gui_controlled_agent_index()
         )
-        lookat = agent_root.translation + mn.Vector3(0, 1, 0)
+        return agent_root.translation, agent_root.rotation
+
+    def _get_camera_lookat_pos(self):
+        agent_pos, _ = self._get_agent_pose()
+        lookat = agent_pos + mn.Vector3(0, 1, 0)
```

Comment:
I know your PR didn't introduce this code, but it would be nice to replace this 1 magic number with a named constant like "pelvis_height" or "lookat_y_offset".

Reply:
addressed